src/main/java/com/jalasoft/schoology/pages/groups/GroupsForm.java

```diff
@@ -31,6 +31,9 @@
     @FindBy(css = "span.group-title")
     private WebElement backToGroupLink;
 
+    @FindBy(css = "#error-page")
```

Comment:
Hi Marco, I noted you added #error-page in EventForm.java and GroupsForm.java, it is being used in an 'if' statement and being called directly in it

I could not find where this FindBy of #error-page is being used.